### ISSUE TITLE
Control-press+control-release + 'a'-keypress  = receive '\x01' in 'on_press' (windows)

### ISSUE THREAD

**Issue by segalion** [moses-palmer/pynput#202]
if I press any key just after I press and release any Control key (left or right), I receive key code as control was pressed on "on_press"

I.e.
Pressing and releasing [control-left], and then pressing 'a' key
- receive '\x01' as key in 'on_press' callback
- write 'a' in notepad
- receive 'a' as key in 'on_release' callback

Next pressig 'a' key works as expected (receive 'a' key in 'on_press')

Enviroment: pynput 1.6 + python3.7 + windows 10 + spanish keyboard

Another question: If I press control+a combination, I receive '\x01'. How can I get 'a' from '\x01' or  '\x01' from 'a'?
Note that crtl+alt+a => receive '<65>' instead of '\x01' in 'on_press' callback

Thanks for all

**Comment by moses-palmer** [comment]
Thank you for your report.

This is actually two issues:
* The state of the modifier key is sticky.
* The key press event value is modified when <kbd>ctrl</kbd> is pressed.

I have pushed a fix to [fixup-win32-listener-modifier-state](https://github.com/moses-palmer/pynput/tree/fixup-win32-listener-modifier-state) for the first one---please test it if you find the time!

The second one is a bit trickier, as what you see printed is the virtual key code reported by _Windows_, and one might argue that is what you should receive in your callback.

For the global hot key feature, this behaviour does not work, so I have added a work-around in [fixup-win32-hotkeys](https://github.com/moses-palmer/pynput/tree/fixup-win32-hotkeys). I am hesitant to use this for all keyboard events however, as that would cause information to be lost.

**Comment by segalion** [comment]
Thanks @moses-palmer 
I can confirm that fixup-win32-listener-modifier-state  works fine with ctrl modifier state.

**Comment by segalion** [comment]
I have tried fixup-win32-hotkeys for the second problem, and I discovered I can get real key with `keyboard_listener._normalize(key)` (get 'a' from key='\x01'). (please, note that seems this fix dont fix the first problem)

As you know, I am working on [bindglobal](https://github.com/segalion/bindglobal) [tkinter-bind-alike](https://effbot.org/tkinterbook/tkinter-events-and-bindings.htm) that replicate the well-know API bind to work with your pynput lib, so I have same problem like you with keys combination like <control-c>

As `keyboard_listener._normalize(key)` return correct key (i.e. 'a' in a <crtl-a> key combination), do you know if I can use this function for all keys in on_press() listener to discover correct combinations (for all keys, for all platfoms)?

Please, could you recomend me how to match all this stuff (vk, scan, char ... ) over the standar bind event atributes?
i.e. 
```
bind('<Control_L-a>', callback)
def callback(event):    
    event.keysym=='\x01' (key) (virtualkey code in windows)
    event.keycode=='a' (_normalize(key))
    event.char=='a' (_normalize(key)) only on char characters
```

Thanks for all this great and impressive work.

**Comment by moses-palmer** [comment]
I introduced `Listener._normalize` to allow implementing the global hotkey feature. Your input makes me think that it could be a public API.

That would require a small amount of work however, as I would need to write tests instead of assuming that if tests of public functions work, the private API is adequate. As I have mentioned in other issues, I no longer have access to any modern Apple system, so that does indeed present a challenge.

As for the suitability of the function, I have only verified that it works for how it is used in the context of the global hotkey listener on *Xorg* and *win32*. Perhaps building `bindglobal` could be implemented on top of this?
